tests/scanner/scanners/external_project_access_scanner_test.py

```diff
@@ -0,0 +1,250 @@
+# Copyright 2018 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for ExternalProjectAccessScanner."""
+from datetime import datetime
+import google.auth
+from google.oauth2 import credentials
+import json
+import unittest
+import mock
+from sqlalchemy.orm import sessionmaker
+
+from google.cloud.forseti.common.util import logger
+from google.cloud.forseti.common.gcp_api import cloud_resource_manager as crm
+from google.cloud.forseti.common.gcp_type import resource_util
+from google.cloud.forseti.common.gcp_type.folder import Folder
+from google.cloud.forseti.common.gcp_type.organization import Organization
+from google.cloud.forseti.common.gcp_type.project import Project
+from google.cloud.forseti.services import db
+from google.cloud.forseti.services.inventory.storage import initialize
+from google.cloud.forseti.services.inventory.storage import Storage
+from google.cloud.forseti.scanner.audit import errors as audit_errors
+from google.cloud.forseti.scanner.scanners import external_project_access_scanner as epas
+
+from tests.common.gcp_api.test_data import fake_crm_responses
+from tests.common.gcp_api.test_data import http_mocks
+from tests.services.api_tests.inventory_test import TestServiceConfig
+from tests.services.inventory.storage_test import ResourceMock
+from tests.services.util.db import create_test_engine
+from tests.unittest_utils import ForsetiTestCase
+from tests.unittest_utils import get_datafile_path
+
+
```

Comment:
Too many blank lines here.  Only 2 blank lines are needed between the import and the global.